.gitignore

```diff
@@ -26,6 +26,7 @@ pip-log.txt
 # sphinx build folder
```

Comment:
I'm confused about the directory structure -- shouldn't these be located underneath `docs/`?